Comp2022/src/main/cpp/subsystems/Vision.cpp

```diff
@@ -22,7 +22,7 @@
 
 // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
 
-Vision::Vision() : m_yfilter(5)
+Vision::Vision() : m_yfilter(5), m_vfilter(5)

```

Comment:
I don't think I understand why we have these as initializers. Are they really necessary?